LocationList.py

```diff
@@ -1835,38 +1835,38 @@ def shop_address(shop_id: int, shelf_id: int) -> int:
     ("Spirit Temple MQ Adult Climb Silver Rupee 4",                  ("SilverRupee", 0x06, (23,0,13), None,                      'Silver Rupee (Spirit Temple Adult Climb)', ("Spirit Temple MQ", "Master Quest", "Silver Rupees"))),
     ("Spirit Temple MQ Adult Climb Silver Rupee 5",                  ("SilverRupee", 0x06, (23,0,14), None,                      'Silver Rupee (Spirit Temple Adult Climb)', ("Spirit Temple MQ", "Master Quest", "Silver Rupees"))),
     # Spirit Temple MQ Pots
-    ("Spirit Temple MQ Lobby Pot 1",                                 ("Pot",          0x06,  (0,0,18), None,                        'Bombs (5)',                             ("Spirit Temple MQ", "Master Quest", "Pots",))),
+    ("Spirit Temple MQ Lobby Pot 1",                                 ("Pot",          0x06,  (0,0,18), None,                        'Recovery Heart',                        ("Spirit Temple MQ", "Master Quest", "Pots",))),
```

Comment:
@rrealmuto the vanilla items seemed out of order basically just for spirit and ganons castle pots. Is it possible they were incorrect, or did something go wrong when I got the actor numbers myself? Thanks!

Reply:
It's very possible I got some wrong between manually updating a bunch of the vanilla items after the fact, and rearranging items in this list. If you are pulling the actor index from the room, and comparing against the flag in LocationList, then I trust you did it right